### DISCUSSION TITLE
Lighthouse Deck is not recieving SteamVR Base Station 2.0 propperly

### DISCUSSION THREAD

**Discussion by GoLut** [bitcraze/discussions#546] [Q&A] [answered]
Hi, I seem to have an issue with my lighthouse deck.

Today I was testing the lighthouse setup using two of the V2 base stations and a Crazyflie 2.1.
I encountered a behavior where at random intervals (~1 to 10 seconds apart) the receiving base station indication would turn off (grey)and then in about a second or so would jump back on again(green). The LEDs on the deck are indicating solid blue and blinking orange indicating receiving sweeps but no GeoData.

I tried resolving the issue by:
- Switching rooms, 
- Changing base station channels, (1-2, 1-3, 1-4  combination)
- Repositioning the base stations (far, close, different angles with respect to the Crazyflie)
- Updating the firmware, python client, and running the default firmware.
All of this did not change the above issue. 

Finally, I swapped out a lighthouse deck for a second board I borrowed.
This board found a connection immediately without issues and I was able to perform the complete geometry estimation.

Is this issue known to the developers or anyone else in the community, could it be a hardware issue or is it software related and fixable?

SIDE NOTE: When I use the saved geometry from the working board it can fly and localize itself however connection to the base stations drops very frequently and the estimation does not seem completely stable. 

I have included the console log, and an image showing this loss of connection after geometry estimation using the borrowed deck, (Base station IDs are 1 and 3)
```
SYS: Build 0:b8d2e0b9e983 (2022.12) CLEAN
SYS: I am 0x203934304D
![Screenshot from 2023-02-02 17-40-25](https://user-images.githubusercontent.com/18073361/216386502-5f14877f-444d-4bb8-b2c7-bf3c3182b247.png)
34500500430018 and I have 1024KB of flash!
CFGBLK: v1, verification [OK]
DECK_CORE: 1 deck(s) found
DECK_CORE: Calling INIT on driver bcLighthouse4 for deck 0
IMU: BMI088: Using I2C interface.
IMU: BMI088 Gyro connection [OK].
IMU: BMI088 Accel connection [OK]
IMU: BMP388 I2C connection [OK]
ESTIMATOR: Using Kalman (2) estimator
CONTROLLER: Using PID (1) controller
MTR-DRV: Using brushed motor driver
SYS: About to run tests in system.c.
SYS: NRF51 version: 2022.12
EEPROM: I2C connection [OK].
STORAGE: Storage check [OK].
IMU: BMI088 gyro self-test [OK]
SYS: Self test passed!
STAB: Wait for sensor calibration...
SYS: Free heap: 19184 bytes
LHFL: Lighthouse bootloader version: 2
LHFL: Bitstream CRC32: 112BC794 [PASS]
LHFL: Firmware version 6 verified, booting deck!
STAB: Ready to fly.
ESTKALMAN: State out of bounds, resetting
```
![Screenshot from 2023-02-02 17-40-25](https://user-images.githubusercontent.com/18073361/216386608-b18acd36-1611-418c-a58f-c43306712e1d.png)

**Comment by krichardsson** [comment, marked as answer]
Hi!

There is (by design) some interference between base stations and they are lost from time to time. Your problem seems to be much worse though and I suspect there is some problem with you deck, the two decks should behave pretty much identically.

Please send us an email at support@bitcraze.io and link to this discussion, we will sort it out through email. Also add a reference to the original order if possible. 

Cheers!
